### PR TITLE
Adding bin/, lib/, include/ to docker build

### PR DESCRIPTION
**Pull request by borisfom**
Signed-off-by: Boris Fomitchev <bfomitchev@nvidia.com>

# Description
Adding installed trtorchc/trtorchexec in bin/ and libraries in lib/

## Type of change

Please delete options that are not relevant and/or add your own.

- New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docker/Dockerfile.21.06 [hunk 1]
```diff
@@ -32,11 +32,19 @@ RUN apt-get update && apt-get install -y locales ninja-build && rm -rf /var/lib/
...
 RUN conda init bash
 
-ENV LD_LIBRARY_PATH /opt/conda/lib/python3.8/site-packages/torch/lib:$LD_LIBRARY_PATH
+WORKDIR /opt/trtorch/src
+RUN mkdir -p /opt/trtorch/lib /opt/trtorch/bin
+RUN cp ./bazel-bin/cpp/bin/trtorchc/trtorchc /opt/trtorch/bin
+RUN cp ./bazel-bin/cpp/api/lib/libtrtorch.so /opt/trtorch/lib
+RUN cp ./bazel-bin/cpp/api/lib/libtrtorchrt.so /opt/trtorch/lib
+RUN cp ./bazel-bin/cpp/api/lib/libtrtorch_plugins.so /opt/trtorch/lib
+
```
Comment (peri044) [inline docker/Dockerfile.21.06:45]: This should be `./bazel-bin/cpp/lib/libtrtorch.so` and other libs as well, otherwise build fails

**Review by peri044 — CHANGES_REQUESTED** [review]
One minor change, otherwise LGTM. 
Although, we could probably extract the `libtrtorch.tar.gz` to copy the libs instead of `bazel-bin` maybe

File: docker/Dockerfile.21.06 (hunk 1 above)
Comment (borisfom) [reply docker/Dockerfile.21.06]: Changed to using tarball - also installing to /usr/local

File: docker/Dockerfile.21.07 [hunk 2]
```diff
@@ -15,14 +15,23 @@ RUN pip install notebook
...
-WORKDIR /opt/trtorch/py
+=======
+
+COPY . /opt/trtorch/src
+WORKDIR /opt/trtorch/src
+RUN cp ./docker/WORKSPACE.cu.docker  WORKSPACE
+
+RUN bazel build //:libtrtorch //:bin --compilation_mode opt
+
+>>>>>>> 76d9d0448adbf4b224b5219a4f5b38602c188cc5
```
Comment (peri044) [inline docker/Dockerfile.21.07:33]: Looks like there are merge conflict changes pending ?

File: docker/Dockerfile.21.07 [hunk 3]
```diff
@@ -32,11 +41,25 @@ RUN apt-get update && apt-get install -y locales ninja-build && rm -rf /var/lib/
...
+=======
+RUN mkdir -p /opt/trtorch/lib /opt/trtorch/bin
+RUN cp ./bazel-bin/cpp/bin/trtorchc/trtorchc /opt/trtorch/bin
+RUN cp ./bazel-bin/cpp/api/lib/libtrtorch.so /opt/trtorch/lib
+RUN cp ./bazel-bin/cpp/api/lib/libtrtorchrt.so /opt/trtorch/lib
+RUN cp ./bazel-bin/cpp/api/lib/libtrtorch_plugins.so /opt/trtorch/lib
+
+ENV LD_LIBRARY_PATH /opt/conda/lib/python3.8/site-packages/torch/lib:/opt/trtorch/lib:${LD_LIBRARY_PATH}
+ENV PATH /opt/trtorch/bin:${PATH}
+>>>>>>> 76d9d0448adbf4b224b5219a4f5b38602c188cc5
```
Comment (peri044) [inline docker/Dockerfile.21.07:62]: merge conflict changes need to be resolved ?

Comment (borisfom) [reply docker/Dockerfile.21.07]: Oups, did not commit the resolve. Should be good now.

**Comment by borisfom** [comment]
I will submit another PR for this.

**Pull request closed without merging**